packages/core/src/utils/image-bitmap.js

```diff
@@ -46,16 +46,16 @@ function exifRotate(img) {
         img.mirror(false, true);
         break;
       case 5: // Mirror horizontal and rotate 270 CW
-        img.rotate(-90, false).mirror(true, false);
```

Comment:
The `false` flag should also be removed under `case 3:` above. I know it doesn't make a difference in case of upside down rotation (180 degrees), but it should still be removed for consistency.

(GitHub doesn't let me comment on that line directly because it's too far from the diff)